src/main/java/com/revature/model/Comment.java

```diff
@@ -37,109 +34,68 @@
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private int id;
 	private Calendar date;
-	private String name;
 	private String text;
+	
 	@ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
-	@JoinColumn(name = "interview_comments")
-	private Interview interviewId;
-
+	@JoinColumn(name = "interview_interviewer")
+	
+	private InterviewInterviewer interviewInterviewer;
+	
 	public Comment() {
 	}
-
-	public Comment(int id, Calendar date, String name, String text, Interview interviewId) {
+	
+	public Comment(int id, Calendar date, String text, InterviewInterviewer interviewInterviewer) {
 		this.id = id;
 		this.date = date;
-		this.name = name;
 		this.text = text;
-		this.interviewId = interviewId;
+		this.interviewInterviewer = interviewInterviewer;
 	}
-
 	public int getId() {
 		return this.id;
 	}
-
 	public void setId(int id) {
 		this.id = id;
 	}
-
 	public Calendar getDate() {
 		return this.date;
 	}
-
 	public void setDate(Calendar date) {
 		this.date = date;
 	}
-
-	public String getName() {
-		return this.name;
-	}
-
-	public void setName(String name) {
-		this.name = name;
-	}
-
 	public String getText() {
 		return this.text;
 	}
-
 	public void setText(String text) {
 		this.text = text;
 	}
 
-	public Interview getInterviewId() {
-		return this.interviewId;
-	}
-
-	public void setInterviewId(Interview interviewId) {
-		this.interviewId = interviewId;
-	}
-
 	public Comment id(int id) {
 		this.id = id;
 		return this;
 	}
-
 	public Comment date(Calendar date) {
 		this.date = date;
 		return this;
 	}
-
-	public Comment name(String name) {
-		this.name = name;
-		return this;
-	}
-
 	public Comment text(String text) {
 		this.text = text;
 		return this;
 	}
 
-	public Comment interviewId(Interview interviewId) {
-		this.interviewId = interviewId;
-		return this;
+	public InterviewInterviewer getInterviewInterviewer() {
+		return interviewInterviewer;
 	}
 
-	@Override
-	public boolean equals(Object o) {
-		if (o == this)
-			return true;
-		if (!(o instanceof Comment)) {
-			return false;
-		}
-		Comment comment = (Comment) o;
-		return id == comment.id && Objects.equals(date, comment.date) && Objects.equals(name, comment.name)
-				&& Objects.equals(text, comment.text) && Objects.equals(interviewId, comment.interviewId);
-	}
-
-	@Override
-	public int hashCode() {
-		return Objects.hash(id, date, name, text, interviewId);
+	public void setInterviewInterviewer(InterviewInterviewer interviewInterviewer) {
+		this.interviewInterviewer = interviewInterviewer;
 	}
 
 	@Override
 	public String toString() {
-		return "{" + " id='" + getId() + "'" + ", date='" + getDate() + "'" + ", name='" + getName() + "'" + ", text='"
-				+ getText() + "'" + ", interviewId='" + getInterviewId() + "'" + "}";
+		return "Comment [id=" + id + ", date=" + date + ", text=" + text + ", interviewInterviewer="
+				+ interviewInterviewer + "]";
 	}
 
-}
+	

```

Comment:
Where are the hashcode and equals methods?

Reply:
Changes are made as per suggestion